interfaces/cython/cantera/reaction.pyx

```diff
@@ -24,32 +29,121 @@ cdef class _ReactionRate:
         will raise an exception.
         """
         if pressure:
-            self.base.update(temperature, pressure)
-            return self.base.eval(temperature, pressure)
+            self.rate.update(temperature, pressure)
+            return self.rate.eval(temperature, pressure)
         else:
-            self.base.update(temperature)
-            return self.base.eval(temperature)
+            self.rate.update(temperature)
+            return self.rate.eval(temperature)
+
+    @staticmethod
+    cdef wrap(shared_ptr[CxxReactionRateBase] rate):
+        """
+        Wrap a C++ Reaction object with a Python object of the correct derived type.
+        """
+        # ensure all reaction types are registered
+        if not(_reaction_rate_class_registry):
+            def register_subclasses(cls):
+                for c in cls.__subclasses__():
+                    rate_type = getattr(c, "_reaction_rate_type")
+                    _reaction_rate_class_registry[rate_type] = c
+                    register_subclasses(c)
+
+            # update global reaction class registry
+            register_subclasses(ReactionRate)
+
+        # identify class
+        rate_type = pystr(rate.get().type())
+        cls = _reaction_rate_class_registry.get(rate_type, ReactionRate)
+
+        # wrap C++ reaction rate
+        cdef ReactionRate rr
+        rr = cls(init=False)
+        rr._rate = rate
+        rr.rate = rr._rate.get()
+        return rr
+
+    @classmethod
+    def from_dict(cls, data, Kinetics kinetics=None):
+        """
+        Create a `ReactionRate` object from a dictionary corresponding to its YAML
+        representation.
+
+        An example for the creation of a ReactionRate from a dictionary is::
+
+            rate = ReactionRate.from_dict(
+                {"rate-constant": {"A": 38.7, "b": 2.7, "Ea": 26191840.0}},
+                kinetics=gas)
+
+        In the example, *gas* is a Kinetics (or Solution) object.
```

Comment:
My preference is to use monospace formatting for function arguments in the docstring, but this is not consistently applied across the codebase. We should again try to develop a standard here, IMO. Also, `Kinetics` and `Solution` can be auto-linked to their docs. Likewise, `ReactionRate` can be linked in the sentence before the example. The same comment applies in `fromYaml`.

Reply:
No problem about changing this to `monospace`. Based on what I had seen, I thought the preference was to use *italics* for variable names.

Reply:
Thank you! Do you have a preference for italics vs. monospace, as one of the maintainers?

Reply:
I don't, honestly. I think monospace is somewhat easier to discern, so I went ahead and replaced all italics in `reaction.pyx`